src/cascade/dismod/db/metadata.py

```diff
@@ -98,21 +99,21 @@ class Node(Base):
 
     node_id = Column(Integer(), primary_key=True)
     node_name = Column(String(), nullable=False, unique=True)
-    parent = Column(Integer, nullable=True)  # Parent is an id in _this_ table.
+    parent = Column(None, ForeignKey("node.node_id"), nullable=True)  # Parent is an id in _this_ table.
```

Comment:
Warning: These can complicate the order of loading the tables. If you have foreign keys, you need to look at the dependency chain and insert tables in order.

Reply:
Yep. But without this you have no guarantees that your keys match. Especially with the possibility of people mucking with tables in pandas outside the DB that seems like something we want to be careful with. But you're right about table order, I haven't made that explicit.